app/models/building.rb

```diff
@@ -20,6 +21,17 @@ class Building < ApplicationRecord
   before_validation :normalize_phone_number
   before_validation :sanitize_description
 
+  def index_image
+    variation =
+      ActiveStorage::Variation.new(Uploads.resize_to_fill(width: 250, height: 190, blob: photo.blob, gravity: "Center"))
```

Comment:
Should we parameterize the width and height somewhere?

Reply:
It's currently this way all over the site. That would be a bigger ticket than this. Can we do it there (in the new ticket) if necessary?

Reply:
Thanks. I'll resolve this.